v22.1/restore.md

```diff
@@ -232,32 +243,41 @@ The examples in this section use the **default** `AUTH=specified` parameter. For
 (3 rows)
 ~~~
 
-When you restore a backup, add the backup's subdirectory path (e.g. `/2021/12/21-142943.73`) to the `RESTORE` statement.
+When you want to [restore a specific backup](#restore-a-specific-backup), add the backup's subdirectory path (e.g., `/2021/12/21-142943.73`) to the `RESTORE` statement. For details on viewing the most recent backup, see [`SHOW BACKUP LATEST`](show-backup.html#show-the-most-recent-backup).
 
-Incremental backups will be appended to the full backup with `BACKUP ... INTO LATEST IN {destination}`. Your storage location will contain the incremental as a date-based subdirectory within the full backup.
+Incremental backups will be appended to the full backup with `BACKUP ... INTO LATEST IN {collection location}`. Your storage location will contain the incremental in the default `/incrementals` directory at the collection location's root. For more detail on collections, see [Backup Collections](take-full-and-incremental-backups.html#backup-collections).
```

Comment:
I don't think it's necessary to include this info in Restore.md as they can look at SHOW BACKUP latest for more on how it displays incremental backups?